src/repos.cpp

```diff
@@ -128,46 +128,26 @@ void RepoManager::enableMultiple(std::initializer_list<std::string> ids)
 
 void RepoManager::disable(const std::string& id) { setEnableState(id, false); }
 
-static std::vector<std::string> getDependenciesEL8(const OS& os)
+static std::vector<std::string> getDependenciesEL(const OS& os, int version)
```

Comment:
There's a class that defines version: https://github.com/viniciusferrao/cloysterhpc/blob/3198cbb5f487963b0f0cb91407836b56a6ccd57d/include/cloysterhpc/os.h#L39